.env.example

```diff
@@ -8,6 +8,15 @@ DB_USER=postgres
 DB_PASSWORD=postgres
 GO_APP_ENV=dev
 OPENAI_KEY=example-api-key
+
+# Rate Limiting Configuration
+RATE_LIMIT_ENABLED=true
+RATE_LIMIT_GLOBAL_RPS=1000
+RATE_LIMIT_API_RPS=100
+RATE_LIMIT_AUTH_RPS=10
+RATE_LIMIT_STORAGE=memory
+# RATE_LIMIT_REDIS_URL=redis://localhost:6379
+# RATE_LIMIT_TRUSTED_PROXIES=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
```

Comment:
Remove RATE_LIMIT_TRUSTED_PROXIES and all its references as it is not implemented anywhere